go/vm/evmc/evmc_interpreter.go

```diff
@@ -224,15 +225,17 @@ func (ctx *hostContext) Selfdestruct(addr evmc.Address, beneficiary evmc.Address
 func (ctx *hostContext) GetTxContext() evmc.TxContext {
 	params := ctx.params
 	return evmc.TxContext{
-		GasPrice:   evmc.Hash(params.GasPrice),
-		Origin:     evmc.Address(params.Origin),
-		Coinbase:   evmc.Address(params.Coinbase),
-		Number:     params.BlockNumber,
-		Timestamp:  params.Timestamp,
-		GasLimit:   int64(params.GasLimit),
-		PrevRandao: evmc.Hash(params.PrevRandao),
-		ChainID:    evmc.Hash(params.ChainID),
-		BaseFee:    evmc.Hash(params.BaseFee),
+		GasPrice:    evmc.Hash(params.GasPrice),
+		Origin:      evmc.Address(params.Origin),
+		Coinbase:    evmc.Address(params.Coinbase),
+		Number:      params.BlockNumber,
+		Timestamp:   params.Timestamp,
+		GasLimit:    int64(params.GasLimit),
+		PrevRandao:  evmc.Hash(params.PrevRandao),
+		ChainID:     evmc.Hash(params.ChainID),
+		BaseFee:     evmc.Hash(params.BaseFee),
+		BlobBaseFee: evmc.Hash(params.BlobBaseFee),
+		BlobHashes:  ctx.EvmcBlobHashes,
```

Comment:
The `./third_party/evmc` submodule needs to be updated to include this field.

Reply:
yes, how should this need for update be reflected ?

Reply:
It is a git submodule. You need to update by
```
cd ./third_party/evmc
git checkout <version you need>
```
and then add it to this commit by
```
cd <tosca directory>
git add ./third_party/evmc
git commit -m "Update evmc dependency"
```

Reply:
ah right, I thought you meant a comment or something else. But before the git submodule change, another PR needs to be merged in the dependency repo.